contrib/components/registry/storage/pvc/registry-deploy-pvc-patch.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Patch
```

Comment:
Is there an API type defined somewhere?

Reply:
No, just reading this from unstructured for now